qiskit_experiments/library/characterization/multi_state_discrimination.py

```diff
@@ -86,23 +87,26 @@ def _default_experiment_options(cls) -> Options:
 
         return options
 
+    @specific_deprecate()
     def __init__(
         self,
-        qubit: int,
+        physical_qubits: Sequence[int],
         backend: Optional[Backend] = None,
         n_states: Optional[int] = None,
         schedules: Optional[Dict[str, ScheduleBlock]] = None,
     ):
         """Setup an experiment to prepare different energy states on a given qubit.
 
         Args:
-            qubit: The qubit on which to run the experiment.
+            physical_qubits: A singleton sequence containing the qubit on which to run the experiment.
```

Comment:
Singleton sequence might be confusing to people not familiar with the terminology—what do you think about "a single-element list containing the qubit..."?